site/layouts/_default/site-header.html

```diff
@@ -10,6 +10,8 @@
             <li class="blog"><a href="/blog" title="Blog Posts">Blog</a></li>
             <li class="community"><a href="/community" title="Community">Community</a></li>
             <li class="resources"><a href="/resources" title="Resources">Resources</a></li>
+            <li class="cli-reference"><a href="/docs/main/cli-reference" title="CLI Reference">CLI
```

Comment:
replace `main` here with {{ .Site.Params.latest }} when next release branch is made with this content.